From 2383aaf470cd419799158d57536d140714af0a43 Mon Sep 17 00:00:00 2001
From: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Date: Wed, 10 Jul 2024 14:15:37 +0200
Subject: [PATCH 1036/1695] dts: stm32: use compatible st,stm32mp215-pinctrl
 for stm32mp21

Use stm32mp21-pinctrl compatibles to be aligned with kernel device
tree.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git c3e3a8b0a395064f530439dd759d878217e668a9 dts: stm32: use compatible st,stm32mp215-pinctrl for stm32mp21]
Change-Id: I391c03f797a851ee6b2cfc42169dd560c0c4901f
Signed-off-by: Thomas Bourgoin <thomas.bourgoin@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/392583
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp211.dtsi | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp211.dtsi b/core/arch/arm/dts/stm32mp211.dtsi
index 1db59e673..730c65e74 100644
--- a/core/arch/arm/dts/stm32mp211.dtsi
+++ b/core/arch/arm/dts/stm32mp211.dtsi
@@ -726,8 +726,8 @@
 		pinctrl: pinctrl@44240000 {
 			#address-cells = <1>;
 			#size-cells = <1>;
-			compatible = "st,stm32mp257-pinctrl";
-			ranges = <0 0x44240000 0xa0400>;
+			compatible = "st,stm32mp215-pinctrl";
+			ranges = <0 0x44240000 0x80400>;
 
 			gpioa: gpio@44240000 {
 				gpio-controller;
@@ -884,10 +884,10 @@
 			};
 		};
 
-		pinctrl_z: pinctrl-z@46200000 {
+		pinctrl_z: pinctrl@46200000 {
 			#address-cells = <1>;
 			#size-cells = <1>;
-			compatible = "st,stm32mp257-z-pinctrl";
+			compatible = "st,stm32mp215-z-pinctrl";
 			ranges = <0 0x46200000 0x400>;
 
 			gpioz: gpio@46200000 {
-- 
2.47.2

